GOVERNANCE.md

```diff
@@ -0,0 +1,150 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the
+`zarr` project, to clarify how decisions are made and how the various
+elements of our community interact.
+
+This is a consensus-based community project. Anyone with an interest in the
+project can join the community, contribute to the project design, and
+participate in the decision making process. This document describes how that
+participation takes place, how to find consensus, and how deadlocks are
+resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The zarr community consists of anyone using or working with the project
+in any way.
+
+## Contributors
+
+A community member can become a contributor by interacting directly with the
+project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification
+  via a GitHub pull request to any https://github.com/zarr-developers repository;
+- reporting a GitHub issue
+  to any https://github.com/zarr-developers repository;
```

Comment:
Any reason there's a linebreak here?

Reply:
Likely to minimize the diff from the previous document.